homeassistant/components/renson/sensor.py

```diff
@@ -0,0 +1,310 @@
+        native_unit_of_measurement="ppm",
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="CO2_HYSTERESIS_FIELD",
+        name="CO2 hysteresis",
+        field=CO2_HYSTERESIS_FIELD,
+        raw_format=False,
+        native_unit_of_measurement="ppm",
+        entity_registry_enabled_default=False,
+    ),
+    RensonSensorEntityDescription(
+        key="BYPASS_TEMPERATURE_FIELD",
+        name="Bypass activation temperature",
+        field=BYPASS_TEMPERATURE_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
+    RensonSensorEntityDescription(
+        key="BYPASS_LEVEL_FIELD",
+        name="Bypass level",
+        field=BYPASS_LEVEL_FIELD,
+        raw_format=False,
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="%",
+    ),
+    RensonSensorEntityDescription(
+        key="FILTER_PRESET_FIELD",
+        name="Filter preset time",
+        field=FILTER_PRESET_FIELD,
+        raw_format=False,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement="days",
+    ),
+)
+
+
+class RensonSensor(RensonEntity, SensorEntity):
+    """Get a sensor data from the Renson API and store it in the state of the class."""
+
+    def __init__(
+        self,
+        description: RensonSensorEntityDescription,
+        api: RensonVentilation,
+        coordinator: RensonCoordinator,
+    ) -> None:
+        """Initialize class."""
+        super().__init__(description.key, api, coordinator)
+
+        self.field = description.field
+        self.entity_description = description
+
+        self._state = None
+        self.data_type = description.field.field_type
+        self.raw_format = description.raw_format
+
+    @property
+    def native_value(self):
+        """Return the value reported by the sensor."""
+        return self._state
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        all_data = self.coordinator.data
+
+        value = self.api.get_field_value(all_data, self.field.name)
+
+        if self.raw_format:
+            self._state = value
+        else:
+            self._state = self.api.parse_value(value, self.data_type)
+
+        self.async_write_ha_state()
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info=None,
+) -> None:
+    """Call the Renson integration to setup."""
+
+    api: RensonVentilation = hass.data[DOMAIN][config_entry.entry_id]["api"]
+    coordinator: RensonCoordinator = hass.data[DOMAIN][config_entry.entry_id][
+        "coordinator"
+    ]
+
+    await coordinator.async_config_entry_first_refresh()
```

Comment:
We already refresh the coordinator when setting up the config entry in the init module.
```suggestion
```